### PR TITLE
Final default is TCP for control

### PR DESCRIPTION
**Pull request by sam-github**
This might be all it takes.

Connect to strongloop/strong-mesh-models#23

### CODE REVIEW

**Comment by sam-github** [comment]
Except there is also the error message, which isn't describing what was being connected to, so that is still pending.

**Comment by kraman** [comment]
LGTM

**Comment by sam-github** [comment]
@kraman please don't lose this in the pmctl refactor! Its the last change I've planned for pm/pmctl in the next week or so.

**Comment by sam-github** [comment]
@crandmck This requires documentation changes.

File: README.md [hunk 1]
```diff
@@ -179,6 +179,8 @@ in this order:
 1. `STRONGLOOP_PM` in environment: may be a local domain path, or an HTTP URL.
 2. `./pmctl`: a process manager running in the current working directory.
 3. `/var/lib/strong-pm/pmctl`: a process manager installed by pm-install.
+4. `/var/lib/strong-pm/pmctl`: a process manager installed by pm-install.
```
Comment (LinusU) [inline README.md]: @sam-github Did you mean for this line to be included twice?

**Pull request merged**